main.css

```diff
@@ -44,7 +44,7 @@ body {
 .header input {
 	display: inline-block !important;
```

Comment:
The indent seems out of place (old and new file), maybe it's a mix of tabs and spaces?

Reply:
Yep, tab and space mix. Fixing now...